demo/src/listen-events.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiPromise, WsProvider } from '@polkadot/api'
```

Comment:
Is there a way to convert this dependency to CORD? 

I wouldn't want our community members to have confusion on when to use which package. As a dependency they will always use the `@cord.network/sdk` package, which internally would have dependency on polkadot etc.

So let the demo scripts also include dependency on @cord.network/sdk itself.

Reply:
This has been addressed - [7c2b974](https://github.com/dhiway/cord.js/pull/220/commits/7c2b974fa323485e813141bfed30862bc52b7307)